api/src/iot/models.py

```diff
@@ -45,3 +45,143 @@ class Device(models.Model):
         to='Person', on_delete=models.SET_NULL, null=True, blank=True, related_name='owner')
     contact = models.ForeignKey(
         to='Person', on_delete=models.SET_NULL, null=True, blank=True, related_name='contact')
+
+
+# These models are loosely based on the data model from sensrnet, the intention
+# is to migrate this data to the sensrnet system once we are able to host our
+# own sensrnet node. For more information about the sensrnet data model see
+# https://kadaster-labs.github.io/sensrnet-home/Model/
+
+
+class Person2(models.Model):
+    """
+    The owner/contact person
+    """
+    # ContactDetails
+    name = models.CharField(
+        max_length=255,
+        verbose_name="Naam (Voornaam [Tussenvoegsel] Achternaam)",
+    )
+    email = models.EmailField(unique=True, verbose_name="E-mail")
+    telephone = models.CharField(max_length=15, verbose_name="Telefoon")
+
+    # LegalEntity
+    organisation = models.CharField(max_length=255, verbose_name="Naam organisatie/bedrijf")
+    website = models.URLField(verbose_name="Website")
+
+    class Meta:
+        verbose_name = 'Eigenaar'
+        verbose_name_plural = 'Eigenaren'
+
+    def __str__(self):
+        return f'{self.email} ({self.name})'
+
+
+class Type2(models.Model):
+    name = models.CharField(max_length=255, verbose_name="Kies soort / type sensor")
+    is_other = models.BooleanField(default=True, verbose_name="Anders, namelijk")
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = 'Type'
+        verbose_name_plural = 'Types'
+
+
+class Theme(models.Model):
+    name = models.CharField(max_length=255, verbose_name="Thema")
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = 'Thema'
+        verbose_name_plural = 'Themas'
+
+
+class LegalGround(models.Model):
+    name = models.CharField(max_length=255, verbose_name="Wettelijke grondslag")
+    link = models.URLField(verbose_name="Privacyverklaring")
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = 'Wettelijke grondslag'
+        verbose_name_plural = 'Wettelijke grondslagen'
+
+
+class Region(models.Model):
+    """
+    A part of Amsterdam city, usually a "Stadsdeel" but users can also enter
+    "other, namely..."
+    """
+    name = models.CharField(max_length=255, verbose_name="Gebied")
+
+
+class Device2(models.Model):
+    """
+    The iot device "thing"
+    """
+    reference = models.CharField(max_length=64, verbose_name="Referentienummer")
```

Comment:
uniek?

Reply:
Ja ik denk dat we de referentie i.c.m. owner uniek moet zijn (er komt ook een bulk melding waar mensen zelf een "unieke" id mogen verzinnen). Er is dus geen garantie dat meerdere eigenaren dezelfde referenties zullen bedenken. Ik dacht dat ik dit al had eigenlijk, maar misschien ben ik dat vergeten te committen

- [ ] uniek op reference + eigenaar